### ISSUE TITLE
[RFC] run script(s) once while test bucket start runs 

### ISSUE THREAD

**Issue by PraveenPenguin** [open-power-host-os/tests#220]
can we have a way that runs a script(s) once when invoking bootstrap and cleanup 
As an example for the dump capture to clean all olds vmcore before starting of the entire test bucket and this should be scenario test case base

**Comment by PraveenPenguin** [comment]
@sathnaga @narasimhan-v thought, please

**Comment by sathnaga** [comment]
@PraveenPenguin Ideally we should not bloat the default bootstrap, and cleanup, the idea of
keeping the env clean for a test start is good, but I feel we can decouple from default run and 
as per my previous suggestion on this PR holds good for this too, https://github.com/open-power-host-os/tests/pull/167#issuecomment-593811967.

It easier to maintain and for usability that way.... one more usecase, manual invocation will silently remove the logs that user might have preserved for a reason...so it is better to decouple.

**Comment by narasimhan-v** [comment]
I feel vmcore managing is not at all avocado-fvt-wrapper's job. We should not try to control anything outside of `what is needed for test to run`

**Comment by PraveenPenguin** [comment]
sure so we can maintain this locally, but RFC is about not only vmcore it just an example how as a user i can leverage fvt-wrapper as i want that trigger point from fvt-wrapper

**Comment by sathnaga** [comment]
@PraveenPenguin user can make use of script plugin and do what ever he/she wants now, I guess after per-test control on script plugin will give more flexibility to user, when it comes to implementing a set of functions to wrapper default, above comments holds good.

**Comment by PraveenPenguin** [comment]
as you know pre/post script is bind with avocado and it will run for every test cases but this RFC proposes the way user can run any command as part of bootstrap and it also user responsibility as his command is valid

**Comment by sathnaga** [comment]
> as you know pre/post script is bind with avocado and it will run for every test cases but this RFC proposes the way user can run any command as part of bootstrap and it also user responsibility as his command is valid 

@PraveenPenguin 
Yes, I know, that's the concern too, if we add it to bootstrap by default it going to cause us trouble untill we solve #213, By any chance you speak about the same  per-test/job granularity of script plugin if then rfc #213 is already  addressing it, we can more details there, if this is different rfc, then I need more details to understand better.
